lib/models/BucketInfo.js

```diff
@@ -147,14 +155,16 @@ class BucketInfo {
         this._deleted = deleted || false;
         this._serverSideEncryption = serverSideEncryption || null;
         this._versioningConfiguration = versioningConfiguration || null;
-        this._locationConstraint = locationConstraint || null;
+        this._locationConstraint = locationConstraint ?
+            locationConstraint.split(zenkoSeparator)[0] : null;
         this._readLocationConstraint = readLocationConstraint || null;
         this._websiteConfiguration = websiteConfiguration || null;
         this._replicationConfiguration = replicationConfiguration || null;
         this._cors = cors || null;
         this._lifecycleConfiguration = lifecycleConfiguration || null;
         this._uid = uid || uuid();
         this._isNFS = isNFS || null;
+        this._ingestion = ingestion ? { Status: 'Enabled' } : null;
```

Comment:
Let's keep this lowercased `{ status: enabled }` should be sufficient.

Reply:
I notice that for other configurations in BucketInfo, they have capitalization like this; would it be better to keep this uniform, or is it still better to go with lowercase?